### PR TITLE
US9 PlayerNodes

### PR DESCRIPTION
**Pull request by toshydev**
- add NodesByOwnerId endpoint with tests, implement owner's nodelist in profile
- remove comment
- add sounds to buttons

### CODE REVIEW

File: backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java [hunk 1]
```diff
@@ -792,8 +793,52 @@ void expectUnauthorizedWhenLoginWithBadCredentials() throws Exception {
...
+
+    @Test
+    @DirtiesContext
+    @WithMockUser(username = "test")
+    void expectNodesOfOwnerWhenGettingNodesByOwnerId() throws Exception {
+        Node node1 = new Node("abc", "123", "Home", 1, 100, new Coordinates(0, 0, 0), 0, 0);
+        Node node2 = new Node("def", "123", "Office", 2, 100, new Coordinates(0, 0, 0), 0, 0);
+        nodeRepo.save(node1);
+        nodeRepo.save(node2);
+        String expected = """
```
Comment (LorTho) [inline backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java:810]: Eventually use of objectMapper to reduce code(jsonStrings)

**Review by LorTho — APPROVED** [review]
looks very nice, some <No newline at end of file>  and one thing in IntegrationTest  with coordinates, i don't know if its important

File: backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java [hunk 2]
```diff
@@ -792,8 +793,52 @@ void expectUnauthorizedWhenLoginWithBadCredentials() throws Exception {
...
+                [
+                    {
+                        "id":"abc",
+                        "name":"Home",
+                        "level":1,
+                        "health":100,
+                        "ownerId": "123",
+                        "coordinates": {
```
[the remark this answers is not in the document]
Comment (toshydev) [reply backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java:818]: the variable name might have been chosen poorly. The coordinate class is actually ONE coordinate.

File: backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java [hunk 3]
```diff
@@ -792,8 +793,52 @@ void expectUnauthorizedWhenLoginWithBadCredentials() throws Exception {
...
     void expectUnauthorizedWhenLoginWithoutUser() throws Exception {
         mockMvc.perform(MockMvcRequestBuilders.post("/api/user/login").with(csrf()))
                 .andExpect(MockMvcResultMatchers.status().isUnauthorized()).andReturn();
     }
+
+    @Test
+    @DirtiesContext
+    @WithMockUser(username = "test")
+    void expectNodesOfOwnerWhenGettingNodesByOwnerId() throws Exception {
+        Node node1 = new Node("abc", "123", "Home", 1, 100, new Coordinates(0, 0, 0), 0, 0);
```
[the remark this answers is not in the document]
Comment (toshydev) [reply backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java:806]: The third one is the timestamp which is not relevant in some tests.

File: backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java (hunk 1 above)
Comment (toshydev) [reply backend/src/test/java/click/snekhome/backend/controller/IntegrationTest.java:810]: Good idea. Never used it but will try it out 👌
Thanks a lot💪

**Pull request merged**